lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -413,6 +414,39 @@ getLocationInfoAsString(const SmallPtrSet<Operation *, 8> &ops) {
   return sstr.str();
 }
 
+/// Emit an optional hardware comment that may exist as an attribute on an
+/// operation.  Return true if a comment was found (and something was written to
+/// the output stream) and false otherwise.
+static bool emitComment(raw_ostream &os, Operation *op) {
+  auto comment =
+      op->getAttrOfType<hw::CommentAttr>(hw::CommentAttr::getMnemonic());
```

Comment:
Please make these be a declared attribute on the ops that support them.

Reply:
Done. `HWModuleOp` now has a new argument `StringAttr comment`.